src/Test/L0/Worker/ExecutionContextL0.cs

```diff
@@ -760,6 +759,7 @@ public void GetExpressionValues_ContainerStepHost()
                 stepHost.Container = containerInfo;
 
                 var ec = new Runner.Worker.ExecutionContext();
+
```

Comment:
can we remove this line? I'm okay with removing the above line too but I don't think this space should exist

Reply:
Done!